client/stylesheets/_responsive.scss

```diff
@@ -229,27 +241,55 @@
 		width: 90%;
 		margin: auto;
 	}
+
     .helpAboutUl li:first-child > p {
     	text-align: left !important;
     }
+
+    .faqdiv h3, .safetydiv h3, .reportdiv h3 {
+        margin-left: 5px !important;
+    }
+
+    .faqdiv ol, .safetydiv ol, .reportdiv ol {
+        margin-left: 10px !important;
+    }
 }
 
 @media (max-width: 380px) {
-    .homeCardLi {
-    	width: 65%;
-    }
-    .searchResultsCardLi {
-    	width: 70% !important;
+    // .homeCardLi {
+    // 	width: 65%;
+    // }
+    // .searchResultsCardLi {
+    // 	width: 70% !important;
+    // }
+    .termsDiv p {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `termsDiv` should be written in lowercase with hyphens